packages/graphql/src/types.ts

```diff
@@ -335,3 +333,14 @@ export interface NestedRecord<T> extends Record<string | symbol | number, T | Ne
 
 /** Input field for graphql-compose */
 export type InputField = { type: string; defaultValue?: string } | string;
+
+export interface Neo4jGraphQLJWTPlugin {
+    rolesPath?: string;
+    secret?: Secret;
```

Comment:
**question (blocking)** Are we sure `secret` should be here? JWKS plugin does not match this interface and it doesn't seem to be used outside the JWT plugin itself

Reply:
Its gone 👀